drivers/tildagon_i2c/tildagon_i2c.c

```diff
@@ -63,7 +63,10 @@ const tca9548a_i2c_mux_t *tildagon_get_i2c_mux() {
 }
 
 void tildagon_i2c_init() {
+    tildagon_i2c_mux.mtx = xSemaphoreCreateMutex();
+    xSemaphoreGive(&tildagon_i2c_mux.mtx);

```

Comment:
This line seems to cause a watchdog timeout

Reply:
you're right! semaphores need to be given first before they are used whereas mutexes don't! I'll fix that in a sec